module5/TaskListApp/TaskListApp/Views/TaskDetailsView.swift

```diff
@@ -31,7 +31,7 @@ struct TaskDetailsView: View {
             Text("Completed: ")
               .bold()
           }
-          .onChange(of: taskCompleted) {
+          .onChange(of: taskCompleted, initial: task.isCompleted) { oldValue, newValue in
```

Comment:
If you're not planning to use a passed in variable, it's customary to replace it with `_` to let future you know that you didn't use it so:
````
.onChange(of: taskCompleted, initial: task.iscompleted) { _, _ in
````
If you wanted to just use one and not the other, you can mix/match variable names and underscores.

Reply:
Good catch, thanks!